lib/matplotlib/patches.py

```diff
@@ -3330,6 +3394,22 @@ def transmute(self, path, mutation_size, linewidth):
                 else:
                     _path.append(Path(verticesA, codesA))
                     _fillable.append(False)
+            elif self.beginarrow_bracket:
+                vertices_list, codes_list = [], []
+                x0, y0 = path.vertices[0]
+                x1, y1 = path.vertices[1]
+                cos_t, sin_t = get_cos_sin(x1, y1, x0, y0)
+                verticesA, codesA = self._get_bracket(x0, y0, cos_t, sin_t,
+                                                      self.widthA * scaleA,
+                                                      self.lengthA * scaleA,
+                                                      self.angleA)
+                vertices_list.append(verticesA)
+                codes_list.append(codesA)
+                vertices = np.concatenate(vertices_list)
+                codes = np.concatenate(codes_list)
```

Comment:
The only thing in these lists is the single output from `self._get_bracket`; I don't know why you need these lists and `concatenate`.